### PR TITLE
OperationMonitor Component

### PR DESCRIPTION
**Pull request by wdo3650**
Adding OperationMonitor Component, a component that outputs determinate progress for file loads, media play progress etc.

### CODE REVIEW

**Comment by wdo3650** [comment]
Per my conversation with Andrew, the name of this component will change.

File: src/components/ProgressIndicator/ProgressIndicator.less [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@ProgressIndicatorMarginBetweenText: 8px;
```
Comment (philworthington) [inline src/components/ProgressIndicator/ProgressIndicator.less]: Add copyright, header comment that we normally use on the top of a less file.

File: src/components/ProgressIndicator/ProgressIndicator.less [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  width: 100%;
+  height: 2px;
+  background-color: @ms-color-neutralLight;
+}
+
+.ms-ProgressIndicator-progressBar {
+  position: absolute;
+  height: 2px;
+  background-color: @ms-color-themePrimary;
+}
```
Comment (philworthington) [inline src/components/ProgressIndicator/ProgressIndicator.less]: Nit: style thing: normally leave an empty line at the end of every file

File: src/components/ProgressIndicator/ProgressIndicator.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. -->
+
+<div class="ms-ProgressIndicator">
+  <div class="ms-ProgressIndicator-itemName">Example.jpg</div>
+  <div class="ms-ProgressIndicator-itemProgress">
+    <div class="ms-ProgressIndicator-progressTrack"></div>
+    <div class="ms-ProgressIndicator-progressBar"></div>
+  </div>
+  <div class="ms-ProgressIndicator-itemDescription">My Progress Description</div>
+</div>
```
Comment (philworthington) [inline src/components/ProgressIndicator/ProgressIndicator.html]: Nit: same as less file: add space at the bottom of the file.

**Comment by philworthington** [comment]
Looks great! Tested and seems to be working really well. Just a few really trivial style things called out.

**Comment by wdo3650** [comment]
Thanks for the suggestions and comments. I made the modifications. Shall I merge?

**Comment by philworthington** [comment]
Approved - looking good to go ahead and merge

**Pull request merged**